reference/conanfile/tools/cmake/cmakedeps.rst

```diff
@@ -215,3 +215,26 @@ Example:
 
         # Generate both MyFileNameConfig.cmake and FindMyFileName.cmake
         self.cpp_info.set_property("cmake_find_mode", "both")
+        
+
+.. _Disable CMakeDeps For Installed CMake imports:
+
+_Disable CMakeDeps For Installed CMake configuration files
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Some projects way want to disable the ``CMakeDeps`` generator for downstream consumers. This can be done by settings ``cmake_find_mode`` to ``none``.
```

Comment:
```suggestion
Some projects may want to disable the ``CMakeDeps`` generator for downstream consumers. This can be done by settings ``cmake_find_mode`` to ``none``.
```